plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -465,6 +486,7 @@ def get_requested_guids(self):
         return requested_guids
 
     def run_telemetry_get_port(self):
+        if self.test_mode: return 9003
```

Comment:
two lines and explanation
and need to use define and not number of port

Reply:
added